File: 11 - Currency Conversion/C/main.c
#include<stdio.h>

int main(void){
    float nEUR, rate;

    printf("How many euros are you exchanging? €");
    scanf("%f", &nEUR);
    printf("What's the exchange rate? ");
    scanf("%f", &rate);

    printf("€%.2f at an exchange rate of %f is £%.2f\n", nEUR, rate, nEUR * rate);

    return 0;
}

File: 11 - Currency Conversion/Golang/main.go
package main

import (
	"fmt"
)

func main() {
	var nEUR, rate float64
	fmt.Printf("How many euros are you exchanging? €")
	fmt.Scanf("%f", &nEUR)

	fmt.Printf("What is the exchange rate? ")
	fmt.Scanf("%f", &rate)

	fmt.Printf("€%.2f at an exchange rate of %f is £%.2f\n", nEUR, rate, nEUR*rate)
}
